src/main/java/accenture/demo/exception/registration/EmailAddressIsAlreadyRegistered.java

```diff
@@ -0,0 +1,8 @@
+package accenture.demo.exception.registration;
+
+public class EmailAddressIsAlreadyRegistered extends RegistrationException {
```

Comment:
Modify class name to EmailAddressIsAlreadyRegisteredException for easier understanding

Reply:
fixed